buildpipeline/pipeline.json

```diff
@@ -278,6 +278,136 @@
             "Platform": "x86",
             "ConfigurationGroup": "Release"
           }
+        },
+        {
+          "Name": "DotNet-CoreFx-Trusted-Windows-UWP",
+          "Parameters": {
+            "PB_Platform": "arm",
+            "PB_BuildArguments": "-framework=uap -buildArch=arm -Release -- /p:SignType=real /p:RuntimeOS=win10",
+            "PB_SyncArguments": "-p -- /p:ArchGroup=arm /p:RuntimeOS=win10",
+            "PB_CreateHelixArguments": "/t:CloudBuild /p:ArchGroup=arm /p:ConfigurationGroup=Release /p:\"EnableCloudTest=false\" /p:\"BuildMoniker=none\" /p:\"TestProduct=corefx\" /p:\"TimeoutInSeconds=1200\" /p:\"TargetOS=Windows_NT\" /p:FilterToOSGroup=Windows_NT"
+          },
+          "ReportingParameters": {
+            "OperatingSystem": "Windows",
+            "Type": "build/product/",
+            "Platform": "arm",
+            "ConfigurationGroup": "Release",
+            "SubType": "Uap"
+          }
+        },
+        {
+          "Name": "DotNet-CoreFx-Trusted-Windows-UWP",
+          "Parameters": {
+            "PB_Platform": "arm64",
+            "PB_BuildArguments": "-framework=uap -buildArch=arm64 -Release -- /p:SignType=real /p:RuntimeOS=win10",
```

Comment:
well my idea was to keep the definitions as close as possible to the Windows one, since once we are at the point were build-tests.cmd works we basically just want to change the definition back and use the same Windows one. This UWP temp one is just because for now I had to disable two steps of the process. If you still feel it would be better to pass this ones as parameters I can change that.